src/ripple/app/tx/impl/SetSignerList.h

```diff
@@ -67,6 +67,12 @@ class SetSignerList : public Transactor
     TER doApply () override;
     void preCompute() override;
 
+    // Interface used by DeleteAccount
+    static
+    TER
+    removeSignersFromLedger (
```

Comment:
Rename this `removeFromLedger` (see comment on `removeDepositPreauthFromLedger`). I will provide a patch in the summary comment.